modules/test/playwright/pages/commerce/commerceLayoutsPage.ts

```diff
@@ -403,7 +410,14 @@ export class CommerceLayoutsPage {
 
 	async createWidgetPage(pageName: string) {
 		await this.addPageButton.first().click();
-		await this.createPageMenuItem.click();
+
+		if (await this.createPageMenuItem.isVisible()) {
+			await this.createPageMenuItem.click();
+		}
+		else {
+			await this.createPublicPageMenuItem.click();
+		}
+
```

Comment:
can you please explain me this? Why sometimes we have use the publicPage and sometimes only page?

Reply:
@smottal This is very strange... I can no longer reproduce that scenario when in some tests it shows just "Page" and in some tests there is "Public Page" and "Private Page".
In light of this, I think we can close this PR.